node-installer/internal/constants/constants.go

```diff
@@ -64,13 +64,15 @@ func KataRuntimeConfig(baseDir string, platform platforms.Platform, debug bool)
 		config.Hypervisor["qemu"]["image"] = filepath.Join(baseDir, "share", "kata-containers.img")
 		config.Hypervisor["qemu"]["kernel"] = filepath.Join(baseDir, "share", "kata-kernel")
 		config.Hypervisor["qemu"]["valid_hypervisor_paths"] = []string{filepath.Join(baseDir, "tdx", "bin", "qemu-system-x86_64")}
+		kernelParams := qemuDmVerityArgs
```

Comment:
```suggestion
		kernelParams, ok := config.Hypervisor["qemu"]["kernel_params"].(string)
		if !ok {
		...
		}
		kernelParams += qemuDmVerityArgs
```

Reply:
We already know that the default value is an empty string, so what's the point in appending? Also, if the default value *wasn't* an empty string, the kernel command line wouldn't be the same we're using to calculate the launch measurement, and so this would cause problems when verifying the attestation reports.

Reply:
Valid, but then let's add a comment that reasons about this.